c2corg_ui/templates/image/filters.html

```diff
@@ -0,0 +1,68 @@
+<%!
+from c2corg_common.attributes import default_langs, activities, image_categories, image_types
+%>
+<%namespace file="../helpers/filters.html" import="add_multiselect"/>
+
+<form app-search-filters app-search-filters-controller-name="appSearchFiltersController">
+  <div class="filters">
+    <div class="row simple-filters">
+
+      <div class="filter">
+        <label translate>date_taken</label><br>
```

Comment:
Rather ``date_time`` as in https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/image.py#L55 so that the same string is used as in the diff page?
Or a human readable string such as ``Date``?